src/Helpers/IPFS.ts

```diff
@@ -0,0 +1,61 @@
+import * as FormData from 'form-data'
+import fetch, { Response } from 'node-fetch'
+import { prop } from 'ramda'
+import * as str from 'string-to-stream'
+
+import { minutesToMiliseconds } from './Time'
+
+const getHash = prop('Hash')
+
+interface IPFSConfiguration {
+  readonly url?: string
+}
+
+interface FetchOptions {
+  readonly timeout?: number
+}
+
+export const IPFS = ({
+  url = 'http://localhost:5001',
+}: IPFSConfiguration) => {
+  const paths = {
+    add: `${url}/api/v0/add`,
+    cat: `${url}/api/v0/cat`,
+  }
+
+  const addText = ({
+    timeout = minutesToMiliseconds(10),
+  }: FetchOptions = {}) => async (text: string): Promise<string> => {
+    const formData = new FormData()
+
+    formData.append('file', str(text), {
+      knownLength: Buffer.from(text).length,
+      filename: 'file',
+      contentType: 'plain/text',
```

Comment:
why are we forcing this to `plain/text`?

Reply:
Not sure if it is necessary, I believe its because this function was created to just add any text. Also out of scope of this PR

https://github.com/poetapp/node/blob/master/src/StorageWriter/IPFS.ts
and
https://github.com/poetapp/node/blob/master/src/StorageReader/IPFS.ts

Reply:
ack